src/Elastic.Markdown/Myst/MarkdownParser.cs

```diff
@@ -121,13 +122,13 @@ private static MarkdownPipeline MinimalPipeline
 	}
 
 	// ReSharper disable once InconsistentNaming
-	private static MarkdownPipeline? PipelineCached;
-	public static MarkdownPipeline Pipeline
+	private MarkdownPipeline? _pipelineCached;
+	public MarkdownPipeline Pipeline
```

Comment:
`MarkdownPipeline` has a lot of internal caches. 

Reusing that as static is key. 

Passing `Build` directly is cleaner but it's worth not doing it for performance reasons.

Reply:
Resolved in chat